bindings/phast.h

```diff
@@ -247,7 +248,6 @@ NAN_METHOD(gasLoadChargeMaterial) {
  * */
     inp = info[0]->ToObject();
     
-    const double thermicReactionType = Get("thermicReactionType");
```

Comment:
Return this line to how it was before. We do not want to call the nan `Get("thermicReactionType")` function directly in the `if` branch. We should use the variable `const double thermicReactionType` in the `if` branch so that it looks like this:
`if (thermicReactionType == 0) {...}`